storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,277 @@
+              <Link href="#" variant="standalone">
+                Rijbewijs aanvragen
+              </Link>
+            </Card>
+          </Grid.Cell>
+        </Grid>
+        <Spotlight color="magenta">
+          <Grid paddingVertical="medium">
+            <Grid.Cell span={{ narrow: 4, medium: 8, wide: 10 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+              <Heading color="inverse" level={3}>
+                {exampleHeading()}
+              </Heading>
+            </Grid.Cell>
+            <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+              <Heading className="ams-mb--sm" color="inverse" level={4}>
+                {exampleHeading()}
+              </Heading>
+              <Paragraph color="inverse">{exampleParagraph()}</Paragraph>
+            </Grid.Cell>
+            <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }}>
+              <Heading className="ams-mb--sm" color="inverse" level={4}>
+                {exampleHeading()}
+              </Heading>
+              <Paragraph color="inverse">{exampleParagraph()}</Paragraph>
+            </Grid.Cell>
+          </Grid>
+        </Spotlight>
+        <Grid paddingVertical="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <Heading className="ams-mb--sm" level={2}>
+              L2 Paragraaf titel
+            </Heading>
+            <Paragraph>{exampleParagraph()}</Paragraph>
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <NavigationColumn sections={burgerzakenLinks[0].slice(3)} />
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }}>
+            <NavigationColumn sections={burgerzakenLinks[1].slice(3)} />
+          </Grid.Cell>
+        </Grid>
+        <Spotlight className="ams-mb--xl" color="green">
+          <Grid paddingVertical="medium">
+            <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+              <Heading className="ams-mb--sm" color="inverse" level={4}>
+                Blijf op de hoogte
+              </Heading>
+              <Paragraph className="ams-mb--md" color="inverse">
+                Schrijf u nu in voor de Nieuwsbrief Amsterdam en ontvang wekelijks nieuws, tips en mooie verhalen over
+                de stad en uw stadsdeel.
+              </Paragraph>
+              <Link color="inverse" href="#" variant="standalone">
+                Ik wil de nieuwsbrief
+              </Link>
+            </Grid.Cell>
+            <Grid.Cell span={{ narrow: 4, medium: 3, wide: 4 }}>
+              <Heading className="ams-mb--sm" color="inverse" level={4}>
+                Deel deze pagina
+              </Heading>
+              <LinkList>
+                <LinkList.Link color="inverse" icon={FacebookIcon}>
```

Comment:
I think it's useful to give these links a `href`, so we can check the tab order.